web/models/github_event.ex

```diff
@@ -0,0 +1,22 @@
+defmodule CodeCorps.GithubEvent do
+  use CodeCorps.Web, :model
+
+  schema "github_events" do
+    field :action, :string
+    field :github_delivery_id, :string
```

Comment:
This is sent as a header with the webhook and should be used for ensuring we don't respond to multiple of the same event. There are no event IDs (!!!), so this is extremely important.

I'm still not quite sure what we're going to do about there not being event IDs. We're going to need some way of determining whether to apply a given change from GitHub to our data, and vice versa.